### PR TITLE
Adding the table for observability pod cpu and memory requests

### PR DESCRIPTION
**Pull request by dockerymick**
https://github.com/open-cluster-management/backlog/issues/8834

### CODE REVIEW

File: observability/observe_intro.adoc [hunk 1]
```diff
@@ -24,6 +24,169 @@ You can customize the observability service by creating custom link:https://prom
...
 For more information about enabling observability, see link:../observability/observability_install.adoc#enable-observability[Enable observability service].
 
+[#observability-pod-capacity-requests]
+=== Observability pod capacity requests
+
+Observability components require 5166mCPU and 11388Mi memory to install the observability service. View the following table of the pod capacity requests that has CPU usage from six managed clusters with `observability-addons` enabled:
+
+.Observability pod capacity requests
+|===
+| Container  | CPU usage (mCPU)  | Memory usage (MiB) | CPU (mCPU) | Memory (Mi) | Replicas | Pod total CPU | Pod total memory
```
Comment (clyang82) [inline observability/observe_intro.adoc]: we do not need have column of CPU usage (mCPU)  and Memory usage (MiB)

Pod total CPU | Pod total memory should be Pod total CPU(mCPU) | Pod total memory(Mi)

File: observability/observe_intro.adoc [hunk 2]
```diff
@@ -24,6 +24,169 @@ You can customize the observability service by creating custom link:https://prom
 
 For more information about enabling observability, see link:../observability/observability_install.adoc#enable-observability[Enable observability service].
 
+[#observability-pod-capacity-requests]
+=== Observability pod capacity requests
+
+Observability components require 5166mCPU and 11388Mi memory to install the observability service. View the following table of the pod capacity requests that has CPU usage from six managed clusters with `observability-addons` enabled:
```
Comment (clyang82) [inline observability/observe_intro.adoc]: > View the following table of the pod capacity requests that has CPU usage from six managed clusters with `observability-addons` enabled

View the following table of the pod capacity requests that is for five managed clusters with `observability-addons` enabled

**Review by clyang82 — CHANGES_REQUESTED** [review]

**Comment by clyang82** [comment]
Thanks @dockerymick for working on this. Just let you know we may change the request capacity to fit for 5 managedcluster, because there is no sufficient resource for observability components running.

File: observability/observe_intro.adoc (hunk 1 above)
Comment (dockerymick) [reply observability/observe_intro.adoc]: Thanks @clyang82 I removed the column

File: observability/observe_intro.adoc (hunk 2 above)
Comment (dockerymick) [reply observability/observe_intro.adoc]: Updated

**Comment by dockerymick** [comment]
@clyang82 Thanks for your review Chun Lin, when you have an opportunity, may you re-review the PR?

**Comment by bjoydeep** [comment]
@dockerymick  we are **re-reviewing** this setting. @subbarao-meduri is leading this effort. He should be focal point now as @clyang82 is out this week. And @subbarao-meduri will be working with @marcolan018 and @clyang82  on this.

**Comment by dockerymick** [comment]
@bjoydeep Thanks for clarifying this!

File: observability/observe_intro.adoc [hunk 3]
```diff
@@ -24,6 +24,152 @@ You can customize the observability service by creating custom link:https://prom
 
 For more information about enabling observability, see link:../observability/observability_install.adoc#enable-observability[Enable observability service].
 
+[#observability-pod-capacity-requests]
+=== Observability pod capacity requests
+
+Observability components require 5166mCPU and 11388Mi memory to install the observability service. View the following table of the pod capacity requests that is for five managed clusters with `observability-addons` enabled:
```
Comment (clyang82) [inline observability/observe_intro.adoc]: s/5166mCPU/3936mCPU

File: observability/observe_intro.adoc [hunk 4]
```diff
@@ -24,6 +24,152 @@ You can customize the observability service by creating custom link:https://prom
...
+| 2.30 GiB
+| 500
+| 512
+| 3
+| 1500
+| 1536
+
+| thanos-rule
+| 67
+| 100
```
Comment (clyang82) [inline observability/observe_intro.adoc]: s/100/50

File: observability/observe_intro.adoc [hunk 5]
```diff
@@ -24,6 +24,152 @@ You can customize the observability service by creating custom link:https://prom
...
+| 512
+| 3
+| 300
```
Comment (clyang82) [inline observability/observe_intro.adoc]: s/300/150

File: observability/observe_intro.adoc [hunk 6]
```diff
@@ -24,6 +24,152 @@ You can customize the observability service by creating custom link:https://prom
...
+| 2
+| 4
+| 25
+| 3
+| 12
+| 75
+
+| memcached
+| 67
+| 100
```
Comment (clyang82) [inline observability/observe_intro.adoc]: s/100/45

File: observability/observe_intro.adoc [hunk 7]
```diff
@@ -24,6 +24,152 @@ You can customize the observability service by creating custom link:https://prom
...
+| 128
+| 3
+| 300
```
Comment (clyang82) [inline observability/observe_intro.adoc]: s/300/135

File: observability/observe_intro.adoc [hunk 8]
```diff
@@ -24,6 +24,152 @@ You can customize the observability service by creating custom link:https://prom
...
+| 384
+
+| exporter
+| 6
+| 10
```
Comment (clyang82) [inline observability/observe_intro.adoc]: s/10/5

File: observability/observe_intro.adoc [hunk 9]
```diff
@@ -24,6 +24,152 @@ You can customize the observability service by creating custom link:https://prom
...
+| 50
+| 3
+| 30
```
Comment (clyang82) [inline observability/observe_intro.adoc]: s/30/15

File: observability/observe_intro.adoc [hunk 10]
```diff
@@ -24,6 +24,152 @@ You can customize the observability service by creating custom link:https://prom
...
+| 150
+
+| thanos-store
+| 694, 228, or 114
+| 500
```
Comment (clyang82) [inline observability/observe_intro.adoc]: s/500/200

File: observability/observe_intro.adoc [hunk 11]
```diff
@@ -24,6 +24,152 @@ You can customize the observability service by creating custom link:https://prom
...
+| 1024
+| 3
+| 1500
```
Comment (clyang82) [inline observability/observe_intro.adoc]: s/1500/600

**Comment by clyang82** [comment]
@dockerymick We have to reduce the cpu requests so that the observability components can be deployed in 3*M4 cluster. Thanks.

File: observability/observe_intro.adoc (hunk 3 above)
Comment (dockerymick) [reply observability/observe_intro.adoc]: updated

File: observability/observe_intro.adoc (hunk 4 above)
Comment (dockerymick) [reply observability/observe_intro.adoc]: updated

File: observability/observe_intro.adoc (hunk 5 above)
Comment (dockerymick) [reply observability/observe_intro.adoc]: Updated

File: observability/observe_intro.adoc (hunk 6 above)
Comment (dockerymick) [reply observability/observe_intro.adoc]: updated

File: observability/observe_intro.adoc (hunk 7 above)
Comment (dockerymick) [reply observability/observe_intro.adoc]: updated

File: observability/observe_intro.adoc (hunk 9 above)
Comment (dockerymick) [reply observability/observe_intro.adoc]: updated

File: observability/observe_intro.adoc (hunk 11 above)
Comment (dockerymick) [reply observability/observe_intro.adoc]: updated

**Comment by dockerymick** [comment]
@clyang82 Thanks for the review, I have addressed your suggestions. When you have an opportunity, may you re-review this PR

**Comment by clyang82** [comment]
@dockerymick we should update the CPU column instead of memory. Do you mind me updating in you PR directly?

**Comment by dockerymick** [comment]
@clyang82 thanks for your patience, I was on PTO. Sure you can update the PR directly

**Comment by dockerymick** [comment]
Based on the changes provided by @clyang82 via slack, we can move forward and merge this PR

**Pull request merged**